### PR TITLE
introduce resolve strategy for paths in the sembench yml

### PR DESCRIPTION
**Pull request by marc-portier**
idea is to 
- [x] introduce a !resolv strategy for postprocessing yml strings so to resolve  '{key}/subpath' entries
- [x] allow Sembench service to be called with a context dict of key --> root-path to be applied in such conversions
- [x] have some SEMBENCH_**_PATH pattern for env variables to be converted to a context dict of key -> root-path
- [x] add tests

### CODE REVIEW

File: .gitignore
Comment (bulricht) [inline .gitignore:1]: please add example.env file

File: tests/resources/resolving-sembench.yml
Comment (bulricht) [inline tests/resources/resolving-sembench.yml:1]: add `args: skip: "keep {one} unresolved"`

File: pysembench/core.py [hunk 1]
```diff
@@ -36,12 +39,22 @@ def on_modified(self, event):
                     os.environ["PYSEMBENCH_WATCHDOG_TIME"] = str(time.time())
 
 
+LOCATION_KEY_PATTERN = r"PYSEMBENCH_(\w+)_PATH"
```
Comment (bulricht) [inline pysembench/core.py]: drop the PY prefix

**Comment by marc-portier** [comment]
requested changes added

**Review by bulricht — APPROVED** [review]

**Pull request merged**